scripts/install-docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export DEBIAN_FRONTEND=noninteractive
```

Comment:
What are these environment variables for?

Reply:
To suppress interactive prompts.  I've added a comment.